ngi_reports/utils/entities.py

```diff
@@ -96,6 +96,7 @@ def __init__(self):
         self.aborted_samples = OrderedDict()
         self.samples = OrderedDict()
         self.flowcells = {}
+        self.proj_lane_info = {}
```

Comment:
Why can't this be included in `self.flowcells`? We are looping through it anyway in `project_summary.md`